src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -39,6 +39,8 @@ FixedwingPositionControl *l1_control::g_control;
 static int _control_task = -1;			///< task handle for sensor task */
 
 FixedwingPositionControl::FixedwingPositionControl() :
+	_sub_airspeed(ORB_ID(airspeed), 0, 0, nullptr),
+	_sub_sensors(ORB_ID(sensor_combined), 0, 0, nullptr),
```

Comment:
It wasn't that long ago that it was using raw accel for this, but it seems like we need to publish a corrected version for quite a few places.